pkg/appStore/AppStoreDeploymentService.go

```diff
@@ -296,12 +297,13 @@ func (impl InstalledAppServiceImpl) updateRequirementDependencies(environment *r
 	if err != nil {
 		return err
 	}
-	chartGitAttrForRequirement := &util.ChartConfig{
+	gitOpsRepoName := impl.chartTemplateService.GetGitOpsRepoName(installedAppVersion.AppStoreApplicationVersion.AppStore.Name)
+	requirmentYamlConfig := &util.ChartConfig{
 		FileName:       appStoreBean.REQUIREMENTS_YAML_FILE,
 		FileContent:    string(requirementDependenciesByte),
 		ChartName:      installedAppVersion.AppStoreApplicationVersion.AppStore.Name,
 		ChartLocation:  argocdAppName,
-		ChartRepoName:  installedAppVersion.AppStoreApplicationVersion.AppStore.Name,
```

Comment:
I am ignoring reviewing this business logic as I don't have context of this story